webapp/src/DOMJudgeBundle/Resources/views/jury/partials/country_select.html.twig

```diff
@@ -0,0 +1,34 @@
+<style>
+    img.countryflag {
+        margin-top: -3px;
+    }
+</style>
```

Comment:
While we're at it, any reason we cannot move this to our global css?

Reply:
@tuupke I think we can?

Reply:
Go for it!